src/gateways/cache.ts

```diff
@@ -40,36 +37,3 @@ export async function scheduleGatewayUpdate(retry = false) {
       : 12 * 60
   });
 }
-
```

Comment:
Moved to `gateway-update-alarm.handler.ts`.

Reply:
👍